stdpopsim/slim_engine.py

```diff
@@ -553,6 +555,39 @@
         }
     }
 }
+
+
+// Save genomic element type information in tree sequence metadata
+// TODO: make this not happen all the time
```

Comment:
Instead of the code in #1068 I decided to output this stuff in the top-level metadata, so I don't have to parse the output to get it - I can pull it from the tree sequence itself.